kubemarine/system.py

```diff
@@ -299,11 +330,11 @@ def generate_etc_hosts_config(inventory, cluster=None):
 
     ignore_ips = list(set(ignore_ips))
 
-    for ip in list(inventory['services']['etc_hosts'].keys()):
+    for ip in list(inventory['services']['etc_hosts_generated'].keys()):
         if len(ip) > max_len_ip:
             max_len_ip = len(ip)
 
-    for ip, names in inventory['services']['etc_hosts'].items():
+    for ip, names in inventory['services']['etc_hosts_generated'].items():
```

Comment:
We no longer need to collect and use ignore_ips collection as etc_hosts_generated cannot have IPs that are going to be removed. In fact, we can have the only method with parameter to generate both configs ` generate_etc_hosts_config('etc_hosts' | 'etc_hosts_generated')`